src/components/MenuIcon/MenuIcon.scss

```diff
@@ -0,0 +1,12 @@
+@import "styles/colors.scss";
+svg {
+    color: $color-off-white;
+    svg:focus {
+        transition: opacity 0.2s ease-in-out;
+        opacity: 0.65;
+    }
+}
+
+ul {
+    background-color: $color-near-black;
```

Comment:
Actually just noticed, I believe the footer is sharing one of these components and I think this is the colour causing it to look different.

Reply:
Not exactly sure what component you're talking about... Are you saying footer has a ul styling which is messing up color?

Reply:
Yeah it was changing the footer colour.